libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1077,6 +1082,89 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info2[] = {
     // @User: Standard
     AP_GROUPINFO("FONT", 4, AP_OSD_Screen, font_index, 0),
 #endif
+
+#if OSD_CRSF_PANELS_ENABLED
+    // @Param: CRSFPWR_EN
+    // @DisplayName: CRSFPWR_EN
+    // @Description: Displays the TX power when using the CRSF RC protocol
+    // @Values: 0:Disabled,1:Enabled
+
+    // @Param: CRSFPWR_X
+    // @DisplayName: CRSFPWR_X
+    // @Description: Horizontal position on screen
+    // @Range: 0 59
+
+    // @Param: CRSFPWR_Y
+    // @DisplayName: CRSFPWR_Y
+    // @Description: Vertical position on screen
+    // @Range: 0 21
+    AP_SUBGROUPINFO(crsf_tx_power, "CRSFPWR", 55, AP_OSD_Screen, AP_OSD_Setting),
```

Comment:
ditto comment on naming

Reply:
I have added an underline/underscore to most of these fields, except the CRSFRSSI panel, as that would make it go above the 16 char limit on OSDx_CRSF_RSSI_EN (this is 17 char long) and create havoc.